metapackages/auto-instrumentations-node/README.md

```diff
@@ -3,7 +3,13 @@
 [![NPM Published Version][npm-img]][npm-url]
 [![Apache License][license-image]][license-url]
 
-This module provides a simple way to initialize multiple Node instrumentations.
+## About
+
+This module provides a way to auto instrument any Node application to capture telemetry from a number of popular libraries and frameworks.
+You can export the telemetry data in a variety of formats. Exporters can be configured via environment variables.
```

Comment:
Users reading this should have some knowledge of what open telemetry is and how it works. So I think "exporting properties/characteristics" will confuse users as it may indicate something else.